hw/ip/hmac/data/hmac.hjson

```diff
@@ -4,11 +4,14 @@
 {
   name:               "hmac",
   human_name:         "HMAC Accelerator",
-  one_line_desc:      "Accelerator for SHA-256-based keyed hash message authentication code and the SHA-256 hash function",
+  one_line_desc:      "Accelerator for SHA-256/384/512-based keyed HMAC and the hash function",
   one_paragraph_desc: '''
-  HMAC Accelerator is a keyed hash based message authentication code generator using [SHA-256][nist-fips-180-4] to check the integrity of an incoming message and optionally a signature signed with the same secret key.
-  This HMAC implementation is not hardened against side-channel analysis (SCA) or fault injection (FI) attacks; it is meant purely for hashing acceleration.
-  If hardened MAC operations are required, either KMAC Accelerator or a software implementation should be used.
+  HMAC Accelerator is a keyed hash based message authentication code generator using
+  [SHA-256/384/512][nist-fips-180-4] to check the integrity of an incoming message and optionally
```

Comment:
A note on consistency: in this file, currently both SHA-256 and SHA2-256 etc. is used, I suggest choosing one of the two and then using it consistently.